Были написаны все тесты, фикстуры и локаторы на функциональность сайта

grayyyyy456 opened the pull request.

grayyyyy456 committed:
Были написаны все тесты, фикстуры и локаторы на функциональность сайта

Ayantea1 commented on tests/test_constructor_click.py:
```diff
@@ -0,0 +1,32 @@
+        button_profile.click()
+        time.sleep(3)
+        WebDriverWait(browser, 10).until(expected_conditions.element_to_be_clickable(Locators.constructor_button)).click()
+        assert browser.current_url == 'https://stellarburgers.nomoreparties.site/'
```
Можно улучшить: Url лучше сохранять в константе

Ayantea1 commented on tests/test_exit.py:
```diff
@@ -0,0 +1,24 @@
+
+class TestClickPersonalAccount:
+    def test_logout_from_personal_account(self, browser):
+        button_login = WebDriverWait(browser, 5).until(expected_conditions.element_to_be_clickable(Locators.login_button))
```
Можно улучшить: повторяющиеся шаги лучше объединять в методы, а методы, использующиеся во многих тестах - выносить в фикстуры

grayyyyy456 committed:
1.Урлы для проверки тестов были присвоены переменным
2.1 Создана фикстура для авторизации на сайте
2.2 Повторяющийся код для тестов с авторизированным пользователям был удален

grayyyyy456 committed:
last commit
